syncstorage/storage/spanner.py

```diff
@@ -358,13 +371,68 @@ def _find_items(self, session, user, collection, **params):
                 bind_types[ts_var] = param_types.TIMESTAMP
 
         # Generate the query
+        fields = params.get("fields")
+        offset_params = params
+
+        if self._bsolm_index_separate and fields is None:
+            # Split the get_items query (TODO: get_item_ids could
+            # reference solely the index) query in 2:
+            # #1: query BsoLastModified directly for bso ids
```

Comment:
What's the intended goal of using this 2 query version? I assume its an optimization to avoid the multiple executions that occur otherwise?

Reply:
Yea, see my description in the issue #118

Reply:
and if this solution looks alright under load testing I'd like to relay those results back to Google, in case they have an alternative or a better explanation for the existing single query's perf